vscode/src/views/TestViewController.ts

```diff
@@ -23,7 +23,7 @@
 
 import { commands, debug, tests, workspace, CancellationToken, TestController, TestItem, TestRunProfileKind, TestRunRequest, Uri, TestRun, TestMessage, Location, Position, MarkdownString } from "vscode";
 import * as path from 'path';
-import { asRange, TestCase, TestSuite } from "../lsp/protocol";
+import { asRange, TestCase as NBTestCase, TestSuite as NbTestSuite } from "../lsp/protocol";
```

Comment:
Is there any particular reason for this name change?
- Logically, I don't think NB should be prefixed, since its not a Netbeans test-case/suite, but interfaces to the LSP protocol.
- The camel-casing becomes tedious and is inconsistent between `NB` and `Nb`.
- A lot of code changes in this PR are just for the renaming and not the fixes, adding a kind of "noise".

As per my understanding, only the changes on lines 38, 238-240 and 246, seem necessary.